migrations/versions/e2be4ab896d3_migrate_old_labels.py

```diff
@@ -0,0 +1,149 @@
+"""migrate old labels
+
+Revision ID: e2be4ab896d3
+Revises: 0b25df40d307
+Create Date: 2019-04-22 12:50:31.062089
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'e2be4ab896d3'
+down_revision = '0b25df40d307'
+
+from datetime import datetime
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import table, column
+
+
+proposal = table('proposal',
+    column('id', sa.Integer()),
+    column('project_id', sa.Integer()),
+    column('technical_level', sa.Unicode(40)),
+    column('session_type', sa.Unicode(40)),
+    column('section_id', sa.Integer()),
+    )
+
+project = table('project',
+    column('id', sa.Integer()),
+    column('name', sa.Unicode(250))
+    )
+
+section = table('section',
+    column('id', sa.Integer()),
+    column('project_id', sa.Integer()),
+    column('name', sa.Unicode(250)),
+    column('title', sa.Unicode(250)),
+    column('description', sa.UnicodeText()),
+    column('created_at', sa.DateTime()),
+    column('updated_at', sa.DateTime())
+    )
+
+label = table('label',
+    column('id', sa.Integer()),
+    column('name', sa.Unicode(250)),
+    column('title', sa.Unicode(250)),
+    column('description', sa.UnicodeText()),
+    column('icon_emoji', sa.UnicodeText()),
+    column('project_id', sa.Integer()),
+    column('main_label_id', sa.Integer()),
+    column('seq', sa.Integer()),
+    column('restricted', sa.Boolean()),
+    column('required', sa.Boolean()),
+    column('archived', sa.Boolean()),
+    column('created_at', sa.DateTime()),
+    column('updated_at', sa.DateTime())
+    )
+
+proposal_label = table('proposal_label',
+    column('proposal_id', sa.Integer()),
+    column('label_id', sa.Integer()),
+    column('created_at', sa.DateTime())
+    )
+
+
+def upgrade():
+    conn = op.get_bind()
+    # Migrate sections -
+    projects = conn.execute(project.select())
+    for proj in projects:
+        sec_count = conn.scalar(
+                sa.select([sa.func.count('*')]).select_from(section).where(section.c.project_id == proj['id'])
+            )
+        if sec_count > 0:
+            # the project has some sections
+            # create section labelset for the project
+            labset = conn.execute(label.insert().values({
```

Comment:
Instead of `labelset`, call this `sections_label`. We'll need the different labels when dealing with errors like with the proposal sections below.